README.md

```diff
@@ -2,30 +2,37 @@
 Tracker to check the covid vaccine slot availability in your district and send mobile notifications through Telegram.
 
 ## Requirements
-Python 3.0+ or Docker must be installed in the local system. 
-  - For Docker: Refer [docker documentation](https://docs.docker.com/engine/install/) to set it locally based on your machine specification.If you are a Windows user and wants to setup docker, follow this [video](https://youtu.be/_9AWYlt86B8)
-  - If you cannot install Docker on your system, install Python by following this [link](https://www.tutorialsteacher.com/python/install-python)
+Anaconda
+  - For Anaconda: Refer [Anaconda](https://docs.anaconda.com/anaconda/install/) to set it locally based on your machine specification.If you are a Windows user and wants to setup Anaconda, follow this [WINDOWS INSTALLATION](https://www.youtube.com/watch?v=syijLJ3oQzU&ab_channel=AmitThinksAmitThinks), for MAC follow [MAC INSTALLATION](https://www.youtube.com/watch?v=V6ZAv7hBH6Y&ab_channel=JustUnderstandingData) and for LINUX [LINUX INSTALLATION](https://www.youtube.com/watch?v=WFswV4J2ZEs&ab_channel=BoostUpStation)
 
 ## How to use?
 There are two parts to this system
 1. Pinging the public COWIN API to get district wise data and checking for availability, every 15 minutes.
-2. Relaying this information to the user's mobile via Telegram Bot. 
-    - For that, you need to create a bot. Follow this [tutorial](https://sendpulse.com/knowledge-base/chatbot/create-telegram-chatbot) to create a telegram bot. Refer the `How to Create a New Bot for Telegram` section. Note the `TOKEN`.
+2. Relaying this information to the user's mobile via Telegram Bot.
+    - Open Anaconda Power Shell Prompt as shown in the image below
+      ![AnancondaPowershell](/images/anaconda_prompt.PNG)
+    - Type in ```pip install telegram-send```
```

Comment:
You don't need these extra packages or steps. If you check the cowin_slot_tracker.py, the connection with the bot is done using the bot endpoint and not through the python package @trishh088